pkg/wifi/iwl.go

```diff
@@ -155,14 +155,16 @@ func (w *IWLWorker) Connect(a ...string) error {
 	// it's been almost instantaneous. But, further, it needs to keep running.
 	go func() {
 		cmd := exec.CommandContext(ctx, "wpa_supplicant", "-i"+w.Interface, "-c/tmp/wifi.conf")
-		cmd.Stdout, cmd.Stderr = os.Stdout, os.Stderr //For an easier time debugging
+		var outBuffer, errBuffer bytes.Buffer
```

Comment:
This was not my suggestion. My suggestion was that your funnction take an io.Writer as a parameter, e.g.
go func(w io.Writer)
and then 
cmd.Stdout, cmd.Stderr = w, w

and the io.Writer can be anything: an os.Pipe, an io.Pipe, a bytes.Buffer, does not matter

Reply:
That makes sense. Would you recommend using a separate io.Writer for each cmd, or can they share the same one?